crates/red_knot_python_semantic/resources/mdtest/binary/integers.md

```diff
@@ -21,23 +21,12 @@ reveal_type(f)  # revealed: Literal[2]
 ## Division by Zero
 
 ```py
-class MyInt(int):
-    def __truediv__(self, other):
-        return 100
-
-def returns_int() -> int:
-    return MyInt(3)
```

Comment:
Maybe we should keep this test, but assert the opposite thing, and paste in your great PR description there as a prose introduction to why we're making the assertion the way we are?